src/config.py

```
@@ -0,0 +1,45 @@
+"""Configuration management for MCP Agent Stack."""
+
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)